frontend/views/pages/GroupSettings.vue

```diff
@@ -61,6 +61,8 @@ page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings
         data-test='saveBtn'
       ) Save changes
 
+  invitelinks
```

Comment:
components usually are kebabe-case in this project. In this case invite-links.

Reply:
Thanks for the info. fixed! :)